### ISSUE TITLE
[Keyboard Navigation – Howdy – Scripts]: “Clone, Edit and Delete” buttons are not accessible using keyboard present in “Scripts” tab.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#10]
**User Experience:**
Users who rely on keyboards will affect to navigate, if Clone, Edit and Delete” buttons are not accessible using keyboard present in “Scripts” tab.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge

**Repro Steps:**
Step 1: Open the above URL in edge browser.
Step 2: Open this URL: https://hcl-szz1925.howdy.ai/dashboard in edge.
Step 3: “Dashboard” page gets opened.
Step 4: Navigate to “Scripts” tab and select it.
Step 5: Navigate to “Clone, Edit and Delete” buttons using keyboard tab key.
Step 6: Verify whether “Clone, Edit and Delete” buttons are accessible using keyboard tab key.

**Actual Result:**
Clone, Edit and Delete” buttons are not accessible using keyboard present in “Scripts” tab.

**Expected Result:**
Clone, Edit and Delete” buttons should be accessible by using keyboard which are present in “Scripts” tab.

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

[MAS2.1.1_Howdy_Scripts.zip](https://github.com/howdyai/howdy-website/files/2973624/MAS2.1.1_Howdy_Scripts.zip)

**Comment by benbrown** [comment]
* Outline styles on focus have been restored to all buttons and links
* Changed "link" buttons to "button" buttons
* Validated that pressing enter triggers button

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed

Now Clone, Edit and Delete buttons are accessible using keyboard.
